Solutions/Corvus.Extensions/Corvus.Extensions.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$(EndjinProjectPropsPath)" Condition="$(EndjinProjectPropsPath) != ''" />
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
This should not be changed. Netstandard2.0 permits use in .NET 4.x, which we still support in a number of places.

Reply:
We've been talking about dropping support for .NET FX and focusing on .NET LTS. Do we have any specific needs or is this more of a "nice to have"?